cron/rotateIndexes.py

```diff
@@ -72,6 +72,14 @@ def esRotateIndexes():
                             logger.debug('do not rotate %s index, month has not changed yet' % index)
                             continue
                     if newindex not in indices:
+                        if 'alerts' in newindex:
+                            logger.debug('Creating %s index with single shard' % newindex)
+                            index_config = {
+                              "settings": {
+                                  "number_of_shards": 1
+                              }
+                            }
+                            es.create_index(newindex, index_config)
```

Comment:
You'll need an else statement here.

Or 'else' we'll be creating the alerts index, then will be trying to create the same index.

Reply:
done

Reply:
You'll want to swap these lines with each other https://github.com/mozilla/MozDef/blob/72bd27d6bae4a585d65e0ffd0de17f7679b424d3/cron/rotateIndexes.py#L82-L83 and fix the indentation so they are both lined up.

You'll also want to swap https://github.com/mozilla/MozDef/blob/72bd27d6bae4a585d65e0ffd0de17f7679b424d3/cron/rotateIndexes.py#L85-L86 as well. TL;DR: log before we perform the action, so if the action goes wrong, the log statement will be accurate.

Reply:
ahh ok, makes sense :)